docs/css/main.css

```diff
@@ -2,6 +2,12 @@
    Author's custom styles
    ========================================================================== */
 
+.language-bash > .highlight > .highlight > code > span.line::before {
```

Comment:
does this trigger the style if it seems a bash code block = triple backticks followed by bash?
is there a way to assert we actually encapsulated all the code in bash code blocks?

Reply:
1.) Yes, this is the style rule for markdown blocks that look like
````
```bash
```
````

2) I don't think there's a "simple" way to enforce people using the correct type of code block fencing. It's really up to the author. The simplest (but also error-prone) way that I can think of is to scan code blocks for keywords like `mvn` `git` `./bin/alluxio`, etc and check if they are fenced properly. I don't really want to do that though.